wp-parsely.php

```diff
@@ -768,6 +768,28 @@ public function insert_parsely_page() {
 		return $parsely_page;
 	}
 
+	/**
+	 * Compare the post_status key against an allowed list (by default, only 'publish'ed content includes tracking data)
+	 *
+	 * @param  int|WP_Post $post  Which post object or id to check.
+	 * @return bool               Should the post status be tracked for the provided post's post_type. By default, only 'publish' is allowed.
+	 *
+	 * @see wp_parsely_track_status_slugs
+	 * @since 2.5.0
```

Comment:
```suggestion
	 * Compare the post_status key against an allowed list (by default, only 'publish'ed content includes tracking data).
	 *
	 * @since 2.5.0
	 *
	 * @param int|WP_Post $post Which post object or ID to check.
	 * @return bool Should the post status be tracked for the provided post's post_type. By default, only 'publish' is allowed.
```